deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -208,3 +209,25 @@ podDisruptionBudget:
   # an integer (e.g. 1) or a percentage value (e.g. 25%).
   # Cannot be used if `minAvailable` is set.
   # maxUnavailable: 1
+
+# +docs:section=CRDs
+# The CRDs installed by this chart are annotated with "helm.sh/resource-policy: keep", this
+# prevents them from being accidentally removed by Helm when this chart is deleted. After
+# deleting the installed chart, the user still has to manually remove the remaining CRDs.
+crds:
+  # -- The 'x-kubernetes-validations' annotation is not supported in Kubernetes 1.22 and below.
+  # This annotation is used by CEL, which is a feature introduced in Kubernetes 1.25 that
+  # improves how validation is performed.
+  # This option allows to force the 'x-kubernetes-validations' annotation to be excluded,
+  # even on Kubernetes 1.25+ clusters.
+  forceRemoveValidationAnnotations: false
+
+# +docs:section=Venafi Connection
+venafiConnection:
+  # -- When set to false, the rendered output does not contain the
+  # VenafiConnection CRDs and RBAC. This is useful for when the
+  # Venafi Connection resources are already installed separately.
+  include: true
+  # -- The name of a VenafiConnection resource which contains the configuration
+  # for authenticating to Venafi.
+  name: venafi-components
```

Comment:
would it make sense to have

```yaml
namespace: venafi
```

too? 

I don't know in what context a user would select a different namespace from the one the Agent is running from, though. But Atanas asked in https://github.com/jetstack/jetstack-secure/pull/552#discussion_r1682728931 to add the ability to configure the namespace in which the venafi connection is located.